### PR TITLE
ChangeWeeks

### PR DESCRIPTION
**Pull request by jeppekikt**
changing weeks both previous and next weeks.

### CODE REVIEW

File: TaskSharper.Calender.WPF/ViewModels/CalendarWeekViewModel.cs [hunk 1]
```diff
@@ -1,45 +1,78 @@
...
+            ReconstructView();
+        }
 
+        private void PrevWeek()
+        {
+            WeeklyOffset--;
+            ReconstructView();
+        }
+        private void ReconstructView()
+        {
```
Comment (alminde7) [inline TaskSharper.Calender.WPF/ViewModels/CalendarWeekViewModel.cs:59]: Should we start the spinner while reconstructing?

File: TaskSharper.Calender.WPF/ViewModels/CalendarWeekViewModel.cs [hunk 2]
```diff
@@ -64,16 +94,19 @@ public Task GetCalelndarEvents()
             //TODO:: Refactor this to make use of observable dictionary
```
Comment (madsmarcussen) [inline TaskSharper.Calender.WPF/ViewModels/CalendarWeekViewModel.cs:95]: Spelling error in method name (Should be GetCalendarEvents()).

**Review by madsmarcussen — CHANGES_REQUESTED** [review]

File: TaskSharper.Calender.WPF/ViewModels/CalendarWeekViewModel.cs (hunk 1 above)
Comment (jeppekikt) [reply TaskSharper.Calender.WPF/ViewModels/CalendarWeekViewModel.cs:59]: fixed

File: TaskSharper.Calender.WPF/ViewModels/CalendarWeekViewModel.cs (hunk 2 above)
Comment (jeppekikt) [reply TaskSharper.Calender.WPF/ViewModels/CalendarWeekViewModel.cs:95]: Fixed

**Pull request merged**